src/main/python/ayab/engine/pattern.py

```diff
@@ -89,8 +92,8 @@ def __convert(self):
             pass
 
         # get palette
-        rgb = self.__pattern.getpalette()[slice(0, 3 * self.__num_colors)]
-        col_array = np.reshape(rgb, (self.__num_colors, 3))
+        rgb = self.__pattern.getpalette()[slice(0, 3 * self.__num_colors)] #type: ignore
```

Comment:
I could NOT for the life of me get this to shake out correctly. Numpy typing really gets too complex for mypy AFAICS